README.md

```diff
@@ -156,12 +157,20 @@ car1.run();   // "Hello! Engine of Camry starting... It has 4 Cylinders."
 car2.run();   // "Hello! Engine of Civic starting... It has 4 Cylinders."
 ```
 
-While on the surface it may look like it does the same thing, `class`es that are _linked_ and *not*
-_copied_ are fundamentally a code reuse mechanism--a way for objects to share code. `Car` looks to
-the parent as a reference, and does not have all `Vehicle`'s properties and behaviors without it.
-The way that the code is shared matters. When you attempt to access a property or method of an object,
-JavaScript will continue searching through the object/`class` and its `prototype` until the end of the
-`prototype` chain is reached. It can create performance and code maintenance issues when misapplied.
+While on the surface it may look like `Vehicle` and `Car` do the same thing, `class`es that are
+_linked_ and *not* _copied_ are fundamentally a code reuse mechanism--a way for objects to share
+code. `Car` looks to the parent as a reference, and does not have all `Vehicle`'s properties and
+behaviors without it. The way that the code is shared matters. When you attempt to access a
+property or method of an object, JavaScript will continue searching through the object/`class`
```

Comment:
take out continue

"When you attempt to access a property or method of an object,
JavaScript searching through the object/`class`..."